crates/rspack_core/src/exports_info.rs

```diff
@@ -1169,9 +1177,14 @@ impl ExportInfo {
     &mut self,
     key: Option<DependencyId>,
     connection: Option<ModuleGraphConnection>,
-    export_name: Option<&Vec<JsWord>>,
+    export_name: Option<&Nullable<Vec<JsWord>>>,
```

Comment:
I am sorry to write code like this, but that's what webpack did.
here `undefined` , `null`, `string[]` has difference semantic.
**undefined**
https://github.com/webpack/webpack/blob/1f99ad6367f2b8a6ef17cce0e058f7a67fb7db18/lib/ExportsInfo.js#L346-L351
**string[]** 
https://github.com/webpack/webpack/blob/1f99ad6367f2b8a6ef17cce0e058f7a67fb7db18/lib/ExportsInfo.js#L313-L318
**null** 
https://github.com/webpack/webpack/blob/1f99ad6367f2b8a6ef17cce0e058f7a67fb7db18/lib/FlagDependencyExportsPlugin.js#L199
https://github.com/webpack/webpack/blob/1f99ad6367f2b8a6ef17cce0e058f7a67fb7db18/lib/Dependency.js#L60-L61

`exportName` will be fallbacked to `[name]` only `fromExport` is equal to `undefined`, https://github.com/webpack/webpack/blob/1f99ad6367f2b8a6ef17cce0e058f7a67fb7db18/lib/FlagDependencyExportsPlugin.js#L247